src/main/scala/com/github/unchama/seichiassist/subsystems/gacha/bukkit/actions/BukkitLotteryOfGachaItems.scala

```diff
@@ -0,0 +1,63 @@
+package com.github.unchama.seichiassist.subsystems.gacha.bukkit.actions
+
+import cats.effect.Sync
+import cats.effect.concurrent.Ref
+import com.github.unchama.seichiassist.subsystems.gacha.application.actions.LotteryOfGachaItems
+import com.github.unchama.seichiassist.subsystems.gacha.domain.{
+  GachaPrize,
+  GachaPrizeId,
+  GachaProbability
+}
+import com.github.unchama.seichiassist.subsystems.gacha.subsystems.gachaprizefactory.bukkit.StaticGachaPrizeFactory
+import org.bukkit.inventory.ItemStack
+
+import scala.annotation.tailrec
+
+class BukkitLotteryOfGachaItems[F[_]: Sync] extends LotteryOfGachaItems[F, ItemStack] {
+
+  import cats.implicits._
+
+  def runLottery(
+    amount: Int,
+    gachaPrizesListRepository: Ref[F, Vector[GachaPrize[ItemStack]]]
+  ): F[Vector[GachaPrize[ItemStack]]] =
+    for {
+      gachaPrizes <- gachaPrizesListRepository.get
+      randomList <-
+        (0 until amount)
+          .map(_ => Sync[F].delay(Math.random()))
+          .toList
+          .traverse(random => random)
```

Comment:
```suggestion
          .map(_ => Sync[F].delay(Math.random()))
          .toList
          .sequence
```

`traverse(identity)`は`sequence`です

----
未来の @KisaragiEffective へ: 間に`IndexedSeq ~> List`相当の変換が挟まっているので`traverse`にはできない。

Reply:
`traverse` にできます